Added a note about our BC promise (#273)
FakerPHP/Faker: README.md

## License

Faker is released under the MIT License. See [`LICENSE`](LICENSE) for details.

## Backward compatibility promise

Faker is using [Semver](https://semver.org/). This means that versions are tagged
with MAJOR.MINOR.PATCH. Only a new major version will be allowed to break backward
compatibility (BC).

> Classes marked as `@experimental` or `@internal` is not included in the BC promise.

Reviewer: ```suggestion
Classes marked as `@experimental` or `@internal` are not included in our backward compatibility promise.
```
